Using Git.md

```diff
@@ -60,6 +61,16 @@ git checkout feature/login-screen
 git merge dev
 ```
 
+### Release branches
+
+Most of the projects have one active stream at the moment. This means that all of the work done by developer can go directly into the dev branch. All the work goes into the next release so there is no need to seperate the dev branch into mulitple release branches. 
```

Comment:
```suggestion
Most projects have one active development stream at the moment. This means that all new changes should be included in the next upcoming release. Because of this, all changes can be merged into the `dev` branch without the need to introduce multiple `release` branches.
```